dev/com.ibm.websphere.security_fat.1/bnd.bnd

```diff
@@ -0,0 +1,37 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+src: \
+  fat/src,\
+  test-applications/WebsphereUserRegistry/src,\
+  test-applications/PasswordUtilAPI/src
+
+fat.project: true
+
+#tested.features: el-4.0, appsecurity-4.0, servlet-5.0, cdi-3.0, jsp-3.0, jaxb-3.0, jaspic-2.0, jacc-2.0
```

Comment:
Is this supposed to be commented out?

Reply:
I can remove it. I had left it in expecting to have to put some in here, but for some reason it didn't require me to. Not quite sure why that is, b/c I can see it using some of the Jakarta features.